PluginAppLoader/apploader.cpp

```diff
@@ -35,15 +35,21 @@ struct CheckPriority {
 
 // Function to parse a string list of ITEM_TYPE
 static QList<int> ParseStringList(const QStringList& l){
-    QList<int> nl;
+    QList<int> result;
     for (const QString& s : l){
         bool ok = false;
-        int v = s.trimmed().toInt(&ok);
-        if (ok && (v != 0) && (v >= -1) && (v < 50)){
-           nl.append(v);
+        int value = s.trimmed().toInt(&ok);
+        if (ok) {
+            if (value == IItem::ITEM_TYPE_ANY) {
```

Comment:
Add a comment that ANY as priority over all other types. In other words, we intentionally want {0, 1, 2, -1, 3} to default to ANY (which would include 4, 5..). Otherwise only allow ANY if the list == {-1}